library/src/commonMain/kotlin/com/gu/recipe/ScaleRecipe.kt

```diff
@@ -83,9 +84,16 @@ internal fun scaleTemplate(template: ParsedTemplate, factor: Float): String {
             }
 
             is TemplateElement.OvenTemperaturePlaceholder -> {
-                var temp = "${element.temperatureC}C"
+                var temp = ""
```

Comment:
Just being curious, do we want string formatting for temperature? why we are not keeping like the way it is written by Editorial?
Example, 
`Preheat the oven to 180C/160C fan/gas mark 4.` 
is it to be something like we need?
`Preheat the oven to 180C (160C fan) gas mark 4.`

Reply:
good 👍  consistent look across recipes and for `F` degree is plus